### PR TITLE
feat: settings indirect edges as false by default

### PR DESCRIPTION
**Pull request by AdiGajbhiye**
### Problem

Describe the problem you are solving. Mention the ticket/issue if applicable.

### Solution

Describe the implemented solution. Add external references if needed.

### Screenshot/Demo

A picture is worth a thousand words. Please highlight the changes if applicable.

### How to test

- Steps to be followed to verify the solution or code changes
- Mention if there is any settings configuration added/changed/deleted

## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

File: src/webview_provider/sqlLineagePanel.ts [hunk 1]
```diff
@@ -309,7 +309,7 @@ export class SQLLineagePanel implements Disposable {
           status: true,
           body: {
             showSelectEdges: config.get("showSelectEdges", true),
-            showNonSelectEdges: config.get("showNonSelectEdges", true),
+            showNonSelectEdges: config.get("showNonSelectEdges", false),
```
Comment (anandgupta42) [inline src/webview_provider/sqlLineagePanel.ts:312]: @AdiGajbhiye Why do we have it in SQLLineage panel?

Comment (AdiGajbhiye) [reply src/webview_provider/sqlLineagePanel.ts:312]: copied by mistake in earlier PRs. not getting used.

Comment (anandgupta42) [reply src/webview_provider/sqlLineagePanel.ts:312]: @AdiGajbhiye should we remove it?

Comment (anandgupta42) [reply src/webview_provider/sqlLineagePanel.ts:312]: or atleast add a todo for now

**Pull request merged**